FlaskWeatherAPI/constants.py

```diff
@@ -0,0 +1,2 @@
+WEATHER_API_KEY = 'VJWG7USK9FMZNP9BB27NUADAK'
```

Comment:
Pro tip: don't commit sensitive information to git for the whole world to see. Because we're friend I won't get you banned using your own key.